values/roundtriptest/int_test.go

```diff
@@ -13,19 +13,27 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestPositiveIntDictionaryEncodeAndDecode(t *testing.T) {
+const (
+	mockDataOutlierMax = 1 << 32
+	mockDataOutlierMin = -(1 << 32)
+)
+
+func TestPositiveIntRawSizeEncodeAndDecode(t *testing.T) {
```

Comment:
Should be `TestPositionVarintEncodeAndDecode`.

Reply:
Changed to `TestPositiveIntVarintEncodeAndDecode`.